src/Concretize.hs

```diff
@@ -386,6 +392,25 @@ typeFromPath env p =
       | otherwise -> error "Local bindings shouldn't be ambiguous."
     Nothing -> error ("Couldn't find " ++ show p ++ " in env:\n" ++ prettyEnvironmentChain env)
 
+-- | Get the mode of a symbol at a given path.
```

Comment:
Perhaps add a comment about that this duplicates some functionality from the Qualify module, as a reminder in the future?

Reply:
You’re right, I added a small comment pointing eager refactorers to the right spot!

Reply:
:D